react/Section/Section.js

```diff
@@ -22,8 +22,8 @@ export default function Section({
   header,
   pullout,
   slim,
-  tone,
-  level,
+  tone = '',
```

Comment:
we don't have neutral style for the section, but it's declared in prop types. Should we remove it from there?

Reply:
Ah yes, I will update that. We have an info style but it was missing from the prop type so I will add that one in